app.js

```diff
@@ -6,4 +67,80 @@ function toggleMenu() {
   menuActive.classList.toggle('active');
   logoActive.classList.toggle('active');
 }
-module.exports = toggleMenu();
\ No newline at end of file
+
+const createPopUp = document.querySelector('.csubmit');
+createPopUp.addEventListener('click', () => {
+  const pop = document.querySelector('body');
+  const section = document.createElement('section');
+  section.className = 'pop-up-section';
+  pop.appendChild(section);
+  section.innerHTML = `
+  <div class="pop-up">
+  <div class="pop-up-box1">
+    <div class="pop-up-box-title">
+      <h1 class="pop-up-title"></h1>
+      <div class="pop-up-x"></div>
+    </div>        
+    <ul class="pop-up-ul">
+      <li class="pop-up-li" id="lenguage1"></li>
+      <li class="pop-up-li" id="lenguage2"></li>
+      <li class="pop-up-li" id="lenguage3"></li>
+    </ul>
+  </div>
+  <div class="pop-up-box2">
+      <div class="pop-up-arrow" id="arrow1"></div>
+    <img class="pop-up-image" src="">
+    <div class="pop-up-secundary-box1">
+      <p class="pop-up-p">
+      </p>
+      <div class="pop-up-secundsry-box2">
+        <a class="pop-up-button" href="https://nimplay.github.io/Microverse_Portfolio/#">
+          <p>See Live</p>
+          <img src="./Images/liveServer.svg" alt="">
+        </a>
+        <a class="pop-up-button" href="https://github.com/nimplay/Microverse_Portfolio">
+          <p>See Source</p>
+          <img src="./Images/whiteGitHub.svg" alt="">
+        </a>              
+      </div>
+    </div>
+    <div class="pop-up-arrow" id="arrow2"></div>
+  </div>
+  </div>
+`;
```

Comment:
- [ ] please make sure that you displayed the cards of the project as in the Figma design right now I see just one 
![image](https://user-images.githubusercontent.com/61790587/173928822-f4ac82f5-1fc0-46a4-9786-52bc51855cd5.png)